src/pybind/mgr/prometheus/module.py

```diff
@@ -759,6 +759,18 @@ def _setup_static_metrics(self) -> Dict[str, Metric]:
             'healthcheck status by type (0=inactive, 1=active)',
             HEALTHCHECK_DETAIL
         )
+        metrics['pool_objects_repaired'] = Metric(
+            'counter',
+            'pool_objects_repaired',
+            'Number of objects repaired in a pool',
+            ('pool_id',)
+        )
+        metrics['daemon_health_metrics'] = Metric(
+            'counter',
```

Comment:
I think @pereman2 also wants to change this to `'gauge'` to test and backport everything in one go.